### PR TITLE
Add the assets command to the vulcan-cli

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR adds a new command to the vulcan-cli that allows to download all the assets, or only the ones with a given asset type, to a local file.

### CODE REVIEW

File: cmd/vulcan-cli/cmd/assets.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	Assets.Flags().BoolVarP(&force, "force", "f", false, "overwrites the output file if it exits")
+	Assets.PersistentFlags().StringArrayVarP(&assetTypes, "type", "", []string{}, assetsHelp)
+	rootCmd.AddCommand(Assets)
+}
+
+func runAssets(args []string, apiClient *cli.CLI) error {
+	path := args[0]
+	if !force {
+		_, err := os.Stat(path)
+		if err != nil && errors.Is(err, fs.ErrExist) {
```
Comment (danfaizer) [inline cmd/vulcan-cli/cmd/assets.go:45]: I believe this condition is not working as is expected by the `-f` flag.

Perhaps is easier:
```
# (if no error, file exists)
jf err == nil {
    return ErrOutputAlreadyExists
}
```
Of course there could be another issues (such as permissions, etc.).

So I would suggest a refactor on this.

**Review by danfaizer — CHANGES_REQUESTED** [review]

File: cmd/vulcan-cli/cmd/assets.go (hunk 1 above)
Comment (manelmontilla) [reply cmd/vulcan-cli/cmd/assets.go]: Absolutely, this is not working at all. 
I'm refactoring out the code that checks if the file exits into its own function, which is what I should have done from the beginning.

**Review by jesusfcr — APPROVED** [review]
Very nice!

**Comment by jesusfcr** [comment]
For next releases I think it would be very useful a json export

```json
[ 
{ "id" "2222", "name":"team-name",
  "assets": [
    {"id":"www.adevinta.com","type":"hostname"},
    {"id":"https://www.adevinta.com","type":"WebAddress"}
  ]
}
]
```

As this will be very flexible to get all the info using jq

```sh
# assets of one team
cat output.json | jq '.[] | select ( .name == 'team' ) | .assets[].id'
...
```

**Pull request merged**